### ISSUE TITLE
Perma-orange: browser_messagemanager_loadprocessscript.js Should get back to 3 processes

### ISSUE THREAD

**Issue by Mardak** [mozilla/activity-stream#2699]
See https://bugzilla.mozilla.org/show_bug.cgi?id=1372664

Not entirely sure what's it's counting, but it might just need a bump from 3 to 4.

```
TEST-UNEXPECTED-FAIL | dom/base/test/browser_messagemanager_loadprocessscript.js | Should get back to 3 processes at this point. - Got 4, expected 3
```
https://treeherder.mozilla.org/logviewer.html#?job_id=106084678&repo=pine&lineNumber=6725

**Comment by k88hudson** [comment]
This is what I'm thinking for fixing this test (bumping to `4` does fix it), but I'm not sure how to handle it in terms of landing the change (in mc? in a diff to be applied later? An if statement on the pref?)

```
diff --git a/dom/base/test/browser_messagemanager_loadprocessscript.js b/dom/base/test/browser_messagemanager_loadprocessscript.js
--- a/dom/base/test/browser_messagemanager_loadprocessscript.js
+++ b/dom/base/test/browser_messagemanager_loadprocessscript.js
@@ -1,5 +1,8 @@
 var ppmm = Cc["@mozilla.org/parentprocessmessagemanager;1"]
            .getService(Ci.nsIMessageBroadcaster);
+
+const BASE_NUMBER_OF_PROCESSES = 3;
+
 ppmm.QueryInterface(Ci.nsIProcessScriptLoader);
 
 function processScript() {
@@ -76,7 +79,7 @@ add_task(function*(){
   }
 
   ppmm.releaseCachedProcesses();
-  is(ppmm.childCount, 3, "Should get back to 3 processes at this point.");
+  is(ppmm.childCount, BASE_NUMBER_OF_PROCESSES, `Should be back to ${BASE_NUMBER_OF_PROCESSES} processes at this point`);
 })
 
 // Test that loading a process script loads in all existing processes
@@ -95,7 +98,7 @@ add_task(function*() {
   if (!gMultiProcessBrowser)
     return;
 
-  is(ppmm.childCount, 3, "Should be three processes at this point");
+  is(ppmm.childCount, BASE_NUMBER_OF_PROCESSES, `Should be ${BASE_NUMBER_OF_PROCESSES} processes at this point`);
 
   // Load something in the main process
   gBrowser.selectedBrowser.loadURI("about:robots");
@@ -125,7 +128,7 @@ add_task(function*() {
     gBrowser.selectedBrowser.loadURI("about:blank");
     yield BrowserTestUtils.browserLoaded(gBrowser.selectedBrowser);
 
-    is(ppmm.childCount, 3, "Should be back to three processes at this point");
+    is(ppmm.childCount, BASE_NUMBER_OF_PROCESSES, `Should be back to ${BASE_NUMBER_OF_PROCESSES} processes at this point`);
 
     // The new process should have responded
     yield check;
```

**Comment by k88hudson** [comment]
Ok, I filed a bug with a possible approach and asked for some feedback here: https://bugzilla.mozilla.org/show_bug.cgi?id=1372664

**Comment by Mardak** [comment]
Will need to be addressed later now that content process is being backed out in #2756 but will need to be fixed to turn it back on for #2777.

**Comment by Mardak** [comment]
https://hg.mozilla.org/mozilla-central/rev/8c54e7d29632
https://bugzilla.mozilla.org/show_bug.cgi?id=1372664
